README.md

```diff
@@ -1,2 +1,48 @@
-# sdk
-JavaScript SDK for connecting to Kontist
+# Kontist SDK
+
+JavaScript SDK for connecting to Kontist using OAuth2 and GraphQL.
+
+## Installation
+
+Add as dependency to your project:
+
+```bash
+npm install kontist-sdk
+```
+
+or
+
+```bash
+yarn add kontist-sdk
+```
+
+## Usage
+
+```typescript
+import { Client } from "kontist-sdk";
+
+const client = new Client({
+  clientId: "YOUR_CLIENT_ID",
+  redirectUri: "YOUR_REDIRECT_URI",
+  scopes: ["transactions"]
+});
+
+client.getAuthUri().then(uri => {
+  /* redirect your user to this uri so he can provide his credentials */
+});
+
```

Comment:
Of course this is just an example - but we only should do this if the user is _not_ already logged in.